kubernetes/keycloakrealmimports.keycloak.org-v1.yml

```diff
@@ -1302,6 +1302,8 @@ spec:
                     type: object
                   otpPolicyInitialCounter:
                     type: integer
+                  otpPolicyCodeReusable:
```

Comment:
Not sure about this.

Reply:
TBH, have no clue. Perhaps we can remove for now and update this code later depending on what @vmuzikar has to say.

Reply:
Ok